common/browser.go

```diff
@@ -50,8 +50,7 @@ type Browser struct {
 	// A *Connection is saved to this field, see: connect().
 	conn connection
 
-	contextsMu     sync.RWMutex
```

Comment:
I've done a stress test on this mutex removal change, and it all seems to work still. I think it's safe to remove since as far as i can tell there's no need to sync the `browserContext` between goroutines. `browser.context` is set when `browser.NewContext` is called, and unset when `browser.Close` is called.

Reply:
😱🤞🙃